src/core/eme/index.ts

```diff
@@ -60,63 +61,146 @@ function createMediaKeysObs(
   return castToObservable<MediaKeys>(keySystemAccess.createMediaKeys());
 }
 
+/**
+ * Store licence if persistent
+ * @param {Object} keySystem
+ */
+function handleSessionStorage(
+  keySystem: IKeySystemOption
+){
+  if (keySystem.persistentLicense) {
+    if (keySystem.licenseStorage) {
+      $storedSessions.setStorage(keySystem.licenseStorage);
+    } else {
+      const error = new Error("no license storage found for persistent license.");
+      throw new EncryptedMediaError("INVALID_KEY_SYSTEM", error, true);
+    }
+  }
+}
```

Comment:
We should maybe think of a better function's name here. This is too generic for a little function which can do side-effects.